src/main/java/com/google/api/generator/gapic/composer/Composer.java

```diff
@@ -152,11 +152,9 @@ public static GapicPackageInfo composePackageInfo(GapicContext context) {
     services.forEach(
         s -> {
           if (context.transport() == Transport.REST) {
-            // REST transport tests donot not use mock services.
-          } else if (context.transport() == Transport.GRPC) {
-            clazzes.add(MockServiceClassComposer.instance().generate(context, s));
-            clazzes.add(MockServiceImplClassComposer.instance().generate(context, s));
-          } else if (context.transport() == Transport.GRPC_REST) {
+            // REST transport tests do not use mock services.
+          } else if (context.transport() == Transport.GRPC
```

Comment:
Please consider still keeping these separate, even if both if statemetns are the same - for consistency reasons and cleaner structure (each transport type has its own section).